test_rock/app.py

```diff
@@ -99,12 +99,3 @@ def postgresql_status():
 def get_env():
     """Return environment variables"""
     return jsonify(dict(os.environ))
-
-
-app2 = Flask("app2")
```

Comment:
It looks like some tests are being removed? Is it because we have simplified the design and hence made them redundant?

Reply:
Yes, I have removed the `webserver_wsgi_path` setting and it's now always `app:app` to be aligned with the extension.